avae/evaluate.py

```diff
@@ -169,17 +169,30 @@ def evaluate(
     # visualise latent disentanglement
     if settings.VIS_DIS:
         vis.latent_disentamglement_plot(
-            x_test, vae, device, data_dim, poses=p_test, mode="_eval"
+            batch[0].shape[-data_dim:],
```

Comment:
it took me a bit to understand what was this for, could we define a variable size once (dsize?) and then reuse in all the different function. For readability.

Reply:
This is done now